### DISCUSSION TITLE
launch into fspicker of cwd by default

### DISCUSSION THREAD

**Discussion by juftin** [davep/textual-astview#12] [Ideas]
Just got into this textual app, it's great 👏 . 

I had an idea for a feature: The `file` argument to the CLI would be optional and by default `astare` would launch the TUI and load the `fspicker` widget with the current working directory by default. 

Not sure how difficult that would be to implement given the empty background behind the widget but if you're interested I could probably give it a shot with a PR. 

Thanks again for the awesome tool!

**Comment by davep** [comment]
Thanks, glad you like it. Yeah, the tool is due a bit of a revisit and I've got some free time next week when I was going to pay this and a few other Textual-based apps of mine a tidy up; this seems like an obvious thing to do.

**Comment by davep** [comment]
Ahh.... I couldn't resist a quick look; [v0.8.0](https://github.com/davep/textual-astview/releases/tag/v0.8.0) happened.

I think I want to give it a tidyup still if I get the chance next week, but this should do the job.
